lib/blacklight/request_builders.rb

```diff
@@ -51,6 +51,14 @@ def solr_search_params_logic= logic
       deprecation_deprecate :solr_search_params_logic=
     end
 
+    def search_builder_class
```

Comment:
I wonder if we should push this and the search_params_logic into the blacklight config.